fastestimator/network.py

```diff
@@ -623,7 +624,8 @@ def _subsample_data(self, data: T, n: int) -> T:
 def build(model_fn: Callable[[], Union[Model, List[Model]]],
           optimizer_fn: Union[str, Scheduler, Callable, List[str], List[Callable], List[Scheduler], None],
           weights_path: Union[str, None, List[Union[str, None]]] = None,
-          model_name: Union[str, List[str], None] = None) -> Union[Model, List[Model]]:
+          model_name: Union[str, List[str], None] = None,
+          mixed_precision: bool = False) -> Union[Model, List[Model]]:
```

Comment:
we need a more global way of enforcing mixed precision

Reply:
people might not want mixed precision on global level